src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * The TaskList object contains the task list, and contains operations to modify or get information about the list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Initialize the TaskList object with 0 tasks.
```

Comment:
Maybe using Initialises instead of Initialise will be better to follow the documentation practices